flow/scripts/load.tcl

```diff
@@ -79,14 +79,14 @@ proc run_equivalence_test {} {
     write_eqy_script
 
     if { [file exists  $::env(LOG_DIR)/4_eqy_output] } {
-	exec "/bin/rm -rf $::env(LOG_DIR)/4_eqy_output"
+      eval exec rm -rf $::env(LOG_DIR)/4_eqy_output
```

Comment:
Use `file delete -force $::env(LOG_DIR)/4_eqy_output`
No need to check for existence either. `file delete` doesn't error on nonexistent paths.